src/Activation/ActivationSummary.tsx

```diff
@@ -0,0 +1,174 @@
+import React, { FunctionComponent } from "react"
```

Comment:
@johnschoeman Would you be so kind as to give this file a once-over and lmk your thoughts?